README.yaml

```diff
@@ -0,0 +1,113 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: terraform-aws-ec2-instance
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "APACHE2"
+
+# Canonical GitHub repo
+github_repo: cloudposse/terraform-aws-ec2-instance
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/terraform-aws-ec2-instance.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/terraform-aws-ec2-instance"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/terraform-aws-ec2-instance.svg"
+    url: "https://github.com/cloudposse/terraform-aws-ec2-instance/releases/latest"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+# Short description of this project
+description: |-
+  Terraform Module for provisioning a general purpose EC2 host.
+
+  Included features:
+  * Automatically create a Security Group
+  * Option to switch EIP attachment
+  * CloudWatch monitoring and automatic reboot if instance hangs
+  * Assume Role capability
+
+# How to use this project
+usage: |-
+  Note: add `${var.ssh_key_pair}` private key to the `ssh agent`.
+
+  Include this repository as a module in your existing terraform code.
+
+  ### Simple example:
+
+  ```hcl
+  module "instance" {
+    source                      = "git::https://github.com/cloudposse/terraform-aws-ec2-instance.git?ref=master"
+    ssh_key_pair                = "${var.ssh_key_pair}"
+    instance_type               = "${var.instance_type}"
+    vpc_id                      = "${var.vpc_id}"
+    security_groups             = ["${var.security_groups}"]
+    subnet                      = "${var.subnet}"
+    name                        = "${var.name}"
+    namespace                   = "${var.namespace}"
+    stage                       = "${var.stage}"
+  }
+  ```
+
+  ### Example with additional volumes and EIP
+
+  ```hcl
+  module "kafka_instance" {
+    source                      = "git::https://github.com/cloudposse/terraform-aws-ec2-instance.git?ref=master"
+    ssh_key_pair                = "${var.ssh_key_pair}"
+    vpc_id                      = "${var.vpc_id}"
+    security_groups             = ["${var.security_groups}"]
+    subnet                      = "${var.subnet}"
+    associate_public_ip_address = "true"
+    name                        = "kafka"
+    namespace                   = "cp"
+    stage                       = "dev"
+    additional_ips_count        = "1"
+    ebs_volume_count            = "2"
+    allowed_ports               = ["22", "80", "443"]
+  }
+  ```
+
+  This module depends on these modules:
+
+  * [terraform-null-label](https://github.com/cloudposse/terraform-null-label)
+
+# References
```

Comment:
I think this should be `related`

Reply:
Add related for 
- https://github.com/cloudposse/terraform-aws-ec2-ami-backup
- https://github.com/cloudposse/terraform-aws-ec2-ami-snapshot
- https://github.com/cloudposse/terraform-aws-ec2-cloudwatch-sns-alarms
- https://github.com/cloudposse/terraform-aws-ec2-autoscale-group